src/main/java/kitchenpos/order/applicatioin/TableGroupService.java

```diff
@@ -0,0 +1,68 @@
+package kitchenpos.order.applicatioin;
+
+import kitchenpos.order.domain.*;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TableGroupService {
+    private final OrderRepository orderRepository;
+    private final OrderTableRepository orderTableRepository;
+    private final TableGroupRepository tableGroupRepository;
+
+    public TableGroupService(OrderRepository orderRepository, OrderTableRepository orderTableRepository,
+                             TableGroupRepository tableGroupRepository) {
+        this.orderRepository = orderRepository;
+        this.orderTableRepository = orderTableRepository;
+        this.tableGroupRepository = tableGroupRepository;
+    }
+
+    @Transactional
+    public TableGroup create(final TableGroup tableGroup) {
+        final List<OrderTable> orderTables = tableGroup.getOrderTables();
+
+        final List<Long> orderTableIds = tableGroup.getOrderTableIds(orderTables);
```

Comment:
```suggestion
        final List<Long> orderTableIds = tableGroup.getOrderTableIds();
```

`TableGroup` 내에 있는 `List<OrderTable>`을 꺼내서 다시 `TableGroup`으로 넘긴뒤 아이디를 뽑아오고 있네요, 이렇게 하신 이유가 있을까요?

Reply:
도메인인 TableGroup에서 Exception 처리하기 위함입니다.